code/__DEFINES/is_helpers.dm

```diff
@@ -1,10 +1,14 @@
 // simple is_type and similar inline helpers
 
+#if DM_VERSION < 513
 #define islist(L) (istype(L, /list))
+#endif
 
 #define in_range(source, user) (get_dist(source, user) <= 1 && (get_step(source, 0)?:z) == (get_step(user, 0)?:z))
 
-#define ismovableatom(A) (istype(A, /atom/movable))
+#if DM_VERSION < 513
+#define ismovable(A) (istype(A, /atom/movable))
```

Comment:
We don't want the new ismovable probably, make sure it doesnt have the same problem isatom and ismob have

Reply:
what problem

Reply:
what are those problems?

Reply:
oh wait right it was isobj, the native isobj acted like ismovable before so we overrode it. you should probably just check if ismovable works correctly compared to our current version

This is probably fine though